compiler.sh

```diff
@@ -87,8 +87,8 @@ if [[ "$build_framework" == true ]]; then
   rm -rf build
   ##python setup.py --use-cython install
   ##python setup.py --use-cython develop
-  CC=${CC} python setup.py build_ext --inplace --use-cython || exit
```

Comment:
Not sure how often this is used in practice, but may need to account for customized c compiliers for the cython builds...This isn't too had to add back or do in different ways without this here, so I don't see a problem in dropping it right now unless something is really depending on this...

Reply:
I removed it b.c. I thought it was [redundant](https://github.com/NOAA-OWP/t-route/blob/aeb5c56ee720685a60599db3bc0a33deca73ed29/compiler.sh#L22). Maybe i'm wrong?? `CC` is always exported (at the top of the build script). I thought that child processes would inherit the environment variables of the parent? Is that not always the case?